front/app/components/IdeaCards/IdeasWithFiltersSidebar/ResetFiltersButton.tsx

```diff
@@ -4,16 +4,22 @@ import { Button, fontSizes } from '@citizenlab/cl2-component-library';
 
 import { FormattedMessage } from 'utils/cl-intl';
 
-import messages from '../messages';
+import ideaCardsMessages from '../messages';
 
 interface Props {
   onClick: () => void;
+  filtersActive: boolean;
 }
 
-const ResetFiltersButton = ({ onClick }: Props) => {
+const ResetFiltersButton = ({ onClick, filtersActive }: Props) => {
   return (
-    <Button buttonStyle="text" fontSize={`${fontSizes.s}px`} onClick={onClick}>
-      <FormattedMessage {...messages.resetFilters} />
+    <Button
+      onClick={onClick}
+      buttonStyle="text"
+      fontSize={`${fontSizes.s}px`}
+      disabled={!filtersActive}
```

Comment:
Our implementation of disabled (via aria-disabled) actually makes the button still focusable, so we don't need to add an extra message to explain this element to screen readers, IMO.